### ISSUE TITLE
[BUG] launch and popup a window with blank content

### ISSUE THREAD

**Issue by jxs1211** [tiny-craft/tiny-rdm#298]
**Tiny RDM Version**
What version of Tiny RDM are you using?
TinyRDM_1.1.14_mac_intel.dmg

**OS Version**
Which OS and version you launch? (Mac/Windows/Linux)
MacBook Pro (16-inch, 2019)
2.6 GHz 六核Intel Core i7

**Redis Version**
Which version of Redis are you using?
n/a

**Describe the bug**
After I installed the package and launch it, pop up a window with blank content
![image](https://github.com/tiny-craft/tiny-rdm/assets/6358406/d92fe78a-87ce-4c3b-a6b5-3016d8814973)

Steps to Reproduce:

1.
2.

**Comment by tiny-craft** [comment]
Which version of macOS you use? Versions below 10.15 have some compatibility issues that cannot be resolved at this time.

**Comment by jxs1211** [comment]
> Which version of macOS you use? Versions below 10.15 have some compatibility issues that cannot be resolved at this time.

Yes, I'm using 10.15.3 for now, so I need to update to a newer version

**Comment by tiny-craft** [comment]
> > Which version of macOS you use? Versions below 10.15 have some compatibility issues that cannot be resolved at this time.
> 
> Yes, I'm using 10.15.3 for now, so I need to update to a newer version? Mac hints that I can update to 10.15.7, should I update that?

The `monaco editor` doesn't support webkit on macOS 10.15, so there's nothing I can do.

**Comment by jxs1211** [comment]
> > > Which version of macOS you use? Versions below 10.15 have some compatibility issues that cannot be resolved at this time.
> > 
> > 
> > Yes, I'm using 10.15.3 for now, so I need to update to a newer version? Mac hints that I can update to 10.15.7, should I update that?
> 
> The `monaco editor` doesn't support webkit on macOS 10.15, so there's nothing I can do.

It works for now after updated to 14.5
